src/utils/websockethandler.js

```diff
@@ -0,0 +1,73 @@
+const { MessageEmbed } = require("discord.js")
+
+module.exports = SocketMessage
```

Comment:
Do any files require this module?

Reply:
This is legacy from the implementation of websockets, which was too much of a hassle for me to implement for different communities (as i can easily use webhooks). If anyone however wants to use this bot or write it on their own, such as adding it to the [https://github.com/Refactorio/FactorioWebInterface](Refactorio/FactorioWebInterface) project, they can use this.

Reply:
I would add a comment at the top of the file to explain this use case.

Reply:
Addeed